web-extensions/chrome/engine/storage.js

```diff
@@ -130,12 +130,18 @@ Storage.prototype.get = function(keys, cb) {
  *
  */
 Storage.prototype.set = function(dataset, cb) {
-    console.log('Wrapper.set');
+    console.log(`Wrapper.set "${JSON.stringify(dataset)}"`);
```

Comment:
that will become huge very quickly (with the "mapping" which contains all sources)